lib/peppermint_sdk.dart

```diff
@@ -7,7 +7,10 @@ export 'src/models/notification/notif.dart';
 export 'src/models/profile/profile.dart';
 export 'src/peppermint_web3.dart';
 export 'src/peppermint_utility.dart';
+export 'src/peppermint_constants.dart';
```

Comment:
No need to export

Reply:
but I used it in main

Reply:
then import in main, not export in here.